docs/platform-specific-code.md

```diff
@@ -102,3 +102,22 @@ const BigButton = require('./BigButton');
 ```
 
 React Native will automatically pick up the right file based on the running platform.
+
+## Native-specific extensions (i.e. sharing code with NodeJS and Web)
+
+You can also use the `.native.js` extension when a module needs to be shared between NodeJS/Web and React Native but it has no Android/iOS differences. This is specially useful for projects that has common code shared among React Native and ReactJS.
+
+For example, say you have the following files in your project:
+
+```sh
+Container.js # picked up by Webpack, Rollup or any other Web bundler
+Container.native.js # picked up by the React Native bundler for both Android and iOS (Metro)
+```
+
+You can still require it without the `.native` extension, as follows:
+
+```javascript
+const Container = require('./Container');
```

Comment:
Does the `require` matter here?

I'd say we should prefer `import` here:

```javascript
import Container from './Container';
```

Reply:
Thanks for looking into this. I agree that require may not be required here. It was just to make sure the user knows the effect of `.native.js` is the same of `.ios.js`. I can remove it if you think it's unnecessary.

Regarding the `import` vs `require`, I'm just following the rest of the site. I'm happy to work on another PR that updates the whole site to use `import` instead of `require`.

Reply:
Sorry, I was not clear in my previous message.

I think you're right, the example is important here to make sure the user knowns the effect of the extension omission. But, I'd prefer `import` instead of `require` syntax.

For what I've seen, `require` are only used with images and all Javascript imports are done with ES6 `import` syntax.